### PR TITLE
doc: update overview

### PR DESCRIPTION
**Pull request by greg-fer**
Updated overview.md and related pages for the latest version of the UI. NCD-724.

### CODE REVIEW

File: doc/docs/using_ppk_app.md [hunk 1]
```diff
@@ -7,23 +7,15 @@ The Power Profiler Kit II (PPK2) must be connected to your computer and powered
...
-    - If the PPK2 is set up to measure in Source Meter mode (see [Measuring current in Source Meter mode](https://docs.nordicsemi.com/bundle/ug_ppk2/page/UG/ppk/measure_current_source_meter.html)), select **Source meter**.
-
-        !!! note "Note"
-            You can change the voltage output to the Device Under Test (DUT, a manufactured product undergoing testing) by using the slider or typing the required voltage.
-
-1. Click **Start**.
+1. Click **Select Device** (in the top left corner) and select the PPK2 from the list.</br>
+   The Power Profiler app checks if the PPK2 has the required firmware and shows a firmware upgrade dialog if needed.
+1. In the device panel's **Mode**, select [**Source meter**](./overview.md#mode) or [**Ampere meter**](./overview.md#mode).
+1. Select the [**Data logger**](./overview.md#sampling-tabs) or [**Real time**](./overview.md#sampling-tabs) mode.
```
Comment (greg-fer) [inline doc/docs/using_ppk_app.md:13]: @kylebonnici , @ketile , what is the correct name for these tabs? "Sampling modes"?

File: doc/docs/overview.md [hunk 2]
```diff
@@ -3,3 +3,215 @@
...
+
+![Advanced controls for voltage limit](./screenshots/ppk2_advanced_voltage.png "Advanced controls for voltage limit")
+
+Use this control to set the voltage limit for the measurement.
+
+## Feedback tab
+
+The Feedback tab lets you send feedback about nRF Connect Power Profiler application to the application development team.
+
+## About tab
```
Comment (greg-fer) [inline doc/docs/overview.md:204]: @kylebonnici , can we add "Open documentation" button to this tab? Pointing to https://docs.nordicsemi.com/bundle/nrf-connect-ppk ?

Comment (greg-fer) [inline doc/docs/overview.md:204]: Can we also add to the "In this app" shortcut list:

- Select measurement range: SHIFT + LMB and drag mouse
- Scroll measurement chart: ALT + mouse wheel

Also, some entries in the shortcuts are currently duplicated, @kylebonnici .

**Comment by greg-fer** [comment]
@kylebonnici , @ketile , could you please review? Thanks!

File: doc/docs/using_ppk_app.md (hunk 1 above)
Comment (greg-fer) [reply doc/docs/using_ppk_app.md:13]: Using "sampling modes" after discussing with Kyle.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
-All functionality is described in the
-[User Guide](https://infocenter.nordicsemi.com/topic/ug_ppk2/UG/ppk/PPK_user_guide_Intro.html)
-(If you use the first version of the hardware, there is a different
-[User Guide for the PPK 1](https://infocenter.nordicsemi.com/topic/ug_ppk/UG/ppk/PPK_user_guide_Intro.html)).
-
 ![screenshot](resources/screenshot.png)
 
+The application also supports the
+[older, first version of the Power Profiler Kit](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit).
+
```
Comment (kylebonnici) [inline README.md:16]: PPK 1 is deprecated and not supported. Last app version that has know support for this is PPK 3.5.5 with nrf connection for desktop 4.4.0

File: README.md [hunk 4]
```diff
@@ -1,31 +1,34 @@
...
+in the nRF Connect from Desktop documentation.
+
+## Documentation
+
+Read the
+[nRF Connect Power Profiler](https://docs.nordicsemi.com/bundle/nrf-connect-ppk/page/index.html)
+official documentation for information about its UI and features.
+
+(The documentation is valid for PPK2. If you use the first version of the hardware, see the
+[online documentation for PPK1](https://docs.nordicsemi.com/bundle/ug_ppk/page/UG/ppk/PPK_user_guide_Intro.html)).
```
Comment (kylebonnici) [inline README.md:31]: To be removed PPK 1 deprecated

File: doc/docs/overview.md [hunk 5]
```diff
@@ -3,3 +3,229 @@
...
+      Loading a `.ppk` file will automatically convert it to a `.ppk2` file.
+      The new file will be saved next to the original `.ppk` file.
+
+      Support for converting `.ppk` files will be removed in the future.
+
+### Temp disk
+
+![Temp disk data controls](./screenshots/ppk2_temp_disk_data.png "Temp disk data controls")
+
+These options only appear before selecting the device and for the **Data logger** sampling mode.
```
Comment (kylebonnici) [inline doc/docs/overview.md:37]:
```suggestion
These options only appear before selecting the device or when the device is selected and on the **Data logger** sampling mode (tab?). 
```

File: doc/docs/overview.md [hunk 6]
```diff
@@ -3,3 +3,229 @@
...
+#### Data logger parameters
+
+![Sampling parameters for Data logger](./screenshots/ppk2_device_panel_sampling_live.png "Sampling parameters for Data logger")
+
+When sampling in the **Data logger** mode, you can select the amount of samples per second that will be recorded.
+The slider lets you select different values up to 100 000 samples per second (100 kHz).
+You can also specify how long the sampling is supposed to last.
+
+After pressing **Start**, the sampling lasts to the moment the sampling time value is reached.
+If you disable **Auto stop sampling**, the sampling will last until you **Stop** it.
```
Comment (kylebonnici) [inline doc/docs/overview.md:76]: This has been combined in the unit dropdown  with forever option

File: doc/docs/overview.md [hunk 7]
```diff
@@ -3,3 +3,229 @@
...
+
+The following table lists the available display options.
+
+| Option                            | Description                                                                                          |
+|-----------------------------------|------------------------------------------------------------------------------------------------------|
+| **Timestamps**                    | Show or hide the timestamps at the bottom of the sampling chart.                                     |
+| **Relative** and **Absolute**     | Select how the time value of the **Timestamps** is to be displayed: relative to the start of the sampling or using machine time (UTC, adjusted per time zone of the machine settings).<br/><br/>If you [**Load**](#device-panel-sections) an old `.ppk` file, the timestamp value will not be shown because of the lack of metadata.         |
+| **Digital channels**              | Show or hide the [digital channels](#digital-channels) under the sampling chart.                     |
+| **Show Minimap**                  | Toggle to show or hide the navigable minimap under the **Data logger**'s chart. You can use the minimap to inspect the logged sampling results. |
```
Comment (kylebonnici) [inline doc/docs/overview.md:105]:
```suggestion
| **Show Minimap**                  | Toggle to show or hide the navigable minimap under the **Data logger**'s chart. You can use the minimap to inspect the logged sampling results and navigate quickly in time range of the recording. |
```

File: doc/docs/screenshots/ppk2_device_panel_advanced.png
Comment (kylebonnici) [inline doc/docs/screenshots/ppk2_device_panel_advanced.png:1]: Outdated

File: doc/docs/screenshots/ppk2_standard_view_connected.png
Comment (kylebonnici) [inline doc/docs/screenshots/ppk2_standard_view_connected.png:1]: Outdated (Auto Sampling removed)

File: README.md (hunk 4 above)
Comment (greg-fer) [reply README.md:31]: I'll keep this line as it is informative about what the scope of the docs is.

File: doc/docs/overview.md [hunk 8]
```diff
@@ -70,10 +74,10 @@ If you don't, you will be prompted about saving any unsaved sampling data before
 
 When sampling in the **Data logger** mode, you can select the amount of samples per second that will be recorded.
 The slider lets you select different values up to 100 000 samples per second (100 kHz).
-You can also specify how long the sampling is supposed to last.
+You can also specify how long the sampling is supposed to last (**300** seconds by default).
```
Comment (kylebonnici) [inline doc/docs/overview.md:77]: Default is forever i.e. no time limit

File: doc/docs/overview.md [hunk 9]
```diff
@@ -70,10 +74,10 @@ If you don't, you will be prompted about saving any unsaved sampling data before
...
 
 After pressing **Start**, the sampling lasts to the moment the sampling time value is reached.
-If you disable **Auto stop sampling**, the sampling will last until you **Stop** it.
+If you specified the sampling to last **forever**, the sampling will last until you manually **Stop** it.
```
Comment (kylebonnici) [inline doc/docs/overview.md:80]: or disk is full as per the Test Disk Full trigger

**Review by kylebonnici — APPROVED** [review]

**Comment by greg-fer** [comment]
@kylebonnici , can you help me with the red build?

**Pull request merged**